client/src/components/HistoryViewer/HistoryViewerVersion.js

```diff
@@ -110,6 +110,10 @@ HistoryViewerVersion.defaultProps = {
   version: defaultVersion,
 };
 
+function mapStateToProps() {
+  return {};
+}
+
```

Comment:
Let's just pass this in as `null` to remove some of the noise.

Reply:
I had `{}` before and it didn't work, which is why I've moved it to a function. I'll try it again with `null` instead =)